src/transformers/models/pegasus/configuration_pegasus.py

```diff
@@ -167,3 +171,74 @@ def num_attention_heads(self) -> int:
     @property
     def hidden_size(self) -> int:
         return self.d_model
+
+
+class PegasusOnnxConfig(OnnxSeq2SeqConfigWithPast):
```

Comment:
Since Pegasus is based on BART, I think we can copy-paste the config implemented in `configuration_bart.py` and add a `# Copied from transformers.models.bart.configuration_bart.BartOnnxConfig with Bart->Pegasus`

This should handle the past-key values correctly, but please let me know if there's still an issue :)